### PR TITLE
Added a new SQS sink

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: sink/sqs.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	sess := session.Must(session.NewSession())
+	svc := sqs.New(sess)
+
+	return &SQSSink{
+		session:   sess,
+		sqs:       svc,
+		queueName: queueName,
+		groupId:   groupId,
+		stopCh:    make(chan interface{}),
+		putCh:     make(chan []byte, 10000),
```
Comment (jippi) [inline sink/sqs.go]: can we keep this at `1000` like the other sinks?

File: sink/sqs.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	}
+
+	close(s.stopCh)
+}
+
+// Put ..
+func (s *SQSSink) Put(data []byte) error {
+	s.putCh <- data
+
+	log.Infof("[sink/sqs] (%d messages left)", len(s.putCh))
```
Comment (jippi) [inline sink/sqs.go]: can we remove this line? it will be quite spammy and the other sinks don't log here

File: sink/sqs.go [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+			if len(buffer) == 10 {
+				s.batchCh <- buffer
+				buffer = make([][]byte, 0)
+			}
+
+			buffer = append(buffer, data)
+
+		case _ = <-ticker.C:
+
+			// If we had any accumulated messages from the ticker, drop them
```
Comment (jippi) [inline sink/sqs.go]: Can you explain what this logic block here is doing?

File: sink/sqs.go [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+func (s *SQSSink) batch() {
+	buffer := make([][]byte, 0)
+	ticker := time.NewTicker(1 * time.Second)
+
+	for {
+		select {
+		case data := <-s.putCh:
+			if len(buffer) == 10 {
```
Comment (jippi) [inline sink/sqs.go]: does it make sense to change order of the append and len check? 

if we append first, and then check the length, we could potentially save up to 1s (from the ticker) before sending out the data

File: sink/sqs.go (hunk 1 above)
Comment (lorenzo) [reply sink/sqs.go]: sure

File: sink/sqs.go (hunk 3 above)
Comment (lorenzo) [reply sink/sqs.go]: I checked the source, and this is actually not doing anything. The channel is of size 1 anyway

Comment (jippi) [reply sink/sqs.go]: ok, make sense

**Pull request merged**